src/pages/index.js

```diff
@@ -2,9 +2,12 @@
 import Card from "../Components/Card.js";
 import FormValidator from "../Components/FormValidator.js";
 import Section from "../Components/Section.js";
+import Popup from "../Components/Popup.js";
 import PopupWithForm from "../Components/PopupWithForm.js";
 import PopupWithImage from "../Components/PopupWithImage.js";
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change it inside and only call it with `super`. The parent methods are available in any inheritor of the class.

Please, delete from `PopupWithImage` method `close`